controllers/ibmblockcsi_controller.go

```diff
@@ -44,9 +42,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
-	"github.com/IBM/ibm-block-csi-operator/controllers/internal/ibmblockcsi"
+	"github.com/IBM/ibm-block-csi-operator/controllers/internal/CRUtils"
```

Comment:
it seems imports are usually lowercase - any way to make it here as well?

Reply:
now it's better? or do you think I should lowercase the package name also?

Reply:
LGTM